ts/src/bitflex.ts

```diff
@@ -0,0 +1,3678 @@
+                    '-1014': InvalidOrder, // -1014 UNKNOWN_ORDER_COMPOSITION Unsupported order combination.
+                    '-1015': RateLimitExceeded, // -1015 TOO_MANY_ORDERS Reach the rate limit .Please slow down your request speed. Too many new orders. Too many new orders; current limit is %s orders per %s.
+                    '-1016': ExchangeError, // -1016 SERVICE_SHUTTING_DOWN This service is no longer available.
+                    '-1020': ExchangeError, // -1020 UNSUPPORTED_OPERATION This operation is not supported.
+                    '-1021': InvalidNonce, // -1021 INVALID_TIMESTAMP Timestamp for this request is outside of the recvWindow. Timestamp for this request was 1000ms ahead of the server's time. Please check the difference between your local time and server time .
+                    '-1022': AuthenticationError, // {"code":-1022,"msg":"Signature for this request is not valid."}
+                    '-1023': OperationRejected, // {"code":-1023,"msg":"Please set IP whitelist before using API"}
+                    '-1100': BadRequest, // -1100 ILLEGAL_CHARS Illegal characters found in a parameter. Illegal characters found in parameter '%s'; legal range is '%s'.
+                    '-1101': BadRequest, // -1101 TOO_MANY_PARAMETERS Too many parameters sent for this endpoint. Too many parameters; expected '%s' and received '%s' Duplicate values for a parameter detected.
+                    '-1102': BadRequest, // -1102 MANDATORY_PARAM_EMPTY_OR_MALFORMED A mandatory parameter was not sent, was empty/null, or malformed. Mandatory parameter '%s' was not sent, was empty/null, or malformed. Param '%s' or '%s' must be sent, but both were empty/null!
+                    '-1103': BadRequest, // -1103 UNKNOWN_PARAM An unknown parameter was sent. In HBTC Open Api , each request requires at least one parameter. {Timestamp}.
+                    '-1104': BadRequest, // -1104 UNREAD_PARAMETERS Not all sent parameters were read. Not all sent parameters were read; read '%s' parameter(s) but was sent '%s'.
+                    '-1105': BadRequest, // -1105 PARAM_EMPTY A parameter was empty. Parameter '%s' was empty.
+                    '-1106': BadRequest, // -1106 PARAM_NOT_REQUIRED A parameter was sent when not required. Parameter '%s' sent when not required.
+                    '-1111': BadRequest, // -1111 BAD_PRECISION Precision is over the maximum defined for this asset.
+                    '-1112': ExchangeError, // -1112 NO_DEPTH No orders on book for symbol.
+                    '-1114': BadRequest, // -1114 TIF_NOT_REQUIRED TimeInForce parameter sent when not required.
+                    '-1115': BadRequest, // -1115 INVALID_TIF Invalid timeInForce. In the current version, this parameter is either empty or GTC.
+                    '-1116': BadRequest, // -1116 INVALID_ORDER_TYPE Invalid orderType. In the current version , ORDER_TYPE values is LIMIT or MARKET.
```

Comment:
is bitflex sharing the exceptions with binance?

Reply:
Maybe. Some of them a catched during the integration, others are copied from docs